pygmt/tests/test_grdimage.py

```diff
@@ -93,12 +93,40 @@ def test_grdimage_over_dateline(xrgrid):
 
 
 @check_figures_equal()
-def test_grdimage_central_longitude(grid):
+@pytest.mark.parametrize("lon0", [0, 123, 180])
+@pytest.mark.parametrize("proj_type", ["H", "W"])
+def test_grdimage_central_meridians(grid, proj_type, lon0):
     """
-    Test that plotting a grid centred at different longitudes/meridians work.
+    Test that plotting a grid with different central meridians (lon0) using
+    Hammer (H) and Mollweide (W) projection systems work.
     """
     fig_ref = Figure()
-    fig_ref.grdimage("@earth_relief_01d_g", projection="W120/15c", cmap="geo")
+    fig_ref.grdimage(
+        "@earth_relief_01d_g", projection=f"{proj_type}{lon0}/15c", cmap="geo"
+    )
+    fig_test = Figure()
+    fig_test.grdimage(grid, projection=f"{proj_type}{lon0}/15c", cmap="geo")
+    return fig_ref, fig_test
+
+
+# Cylindrical Equidistant (Q) projections plotted with xarray and NetCDF grids
+# are still slightly different with an RMS error of 25, see issue at
+# https://github.com/GenericMappingTools/pygmt/issues/390
+# TO-DO remove tol=1.5 and pytest.mark.xfail once bug is solved in upstream GMT
+@check_figures_equal(tol=1.5)
+@pytest.mark.parametrize("lat0", [0, 30])
+@pytest.mark.parametrize("lon0", [0, 123, 180])
+@pytest.mark.parametrize("proj_type", [pytest.param("Q", marks=pytest.mark.xfail), "S"])
+def test_grdimage_central_meridians_and_standard_parallels(grid, proj_type, lon0, lat0):
```

Comment:
Using `xfail` (again) to temporarily allow the Cylindrical Equidistant (Q) tests to pass (instructions at https://docs.pytest.org/en/latest/skipping.html#skip-xfail-with-parametrize). Note that there are **2 `xpass`**es (because not all projections are wrong), and **4 xfail**s.

Also, I've set the RMS tolerance to 1.5 to allow the General Stereographic (S) projection tests to pass, for reference, here are the plot diffs (along the Greenwich Meridian):

| NetCDF (expected) | Xarray (test) | Diff |
|---|---|---|
| ![test_grdimage_central_meridians_and_standard_parallels S-0-0-png -expected](https://user-images.githubusercontent.com/23487320/92429449-111b5f00-f1e6-11ea-8b49-73b11da0e918.png) | ![test_grdimage_central_meridians_and_standard_parallels S-0-0-png](https://user-images.githubusercontent.com/23487320/92429446-0f519b80-f1e6-11ea-8a2e-cb923da0cb26.png) | ![test_grdimage_central_meridians_and_standard_parallels S-0-0-png -failed-diff](https://user-images.githubusercontent.com/23487320/92429451-11b3f580-f1e6-11ea-98cf-a644fef7b6a4.png) |